demosplan/DemosPlanCoreBundle/Repository/StatementRepository.php

```diff
@@ -192,6 +193,9 @@ public function addObject($statement): Statement
             return $statement;
         } catch (Exception $e) {
             $this->getLogger()->warning('Add StatementObject failed Message: ', [$e]);
+            if (str_contains($e->getMessage(), 'internId_procedure')) {
```

Comment:
I was wondering if the internID is empty, is it throwing this exception or another one?

Reply:
InternId can be null

Reply:
catch (UniqueConstraintViolationException $e) {
            if (str_contains($e->getMessage(), 'internId_procedure')) {
                throw DuplicateInternIdException::create('Eingangsnummer', $statement->getProcedureId());
            }
            throw $e;
        }